cmd/eval-dev-quality/cmd/config.go

```diff
@@ -21,6 +24,14 @@ type ModelsConfiguration struct {
 	Available []string
 }
 
+// RepositoryConfiguration holds repository data of how an evaluation was configured.
+type RepositoryConfiguration struct {
+	// Selected holds the repositories selected for an evaluation.
+	Selected []string
```

Comment:
Is it possible to define which tasks are running for a repository?

Reply:
I guess we should also add the "used" tasks to this list. Then you wouldn't need to cross reference between available/selected if you want to gather metrics.

Reply:
We currently don't have a way to select which tasks are running. If a repository supports tasks, they all run. That also makes the current logic easier. I had the version where I stored the tasks and it was just redundant information, should we really add it?